Python/pywarpx/picmi.py

```diff
@@ -2739,6 +2745,9 @@ class Simulation(picmistandard.PICMI_Simulation):
 
     warpx_used_inputs_file: string, optional
         The name of the text file that the used input parameters is written to,
+
+    warpx_max_dt: float, optional
+        The maximum allowable timestep when using adaptive timestepping (only available for non-electromagnetic solvers)
```

Comment:
Seeing as this is specific to the Poisson solver I think it would make more sense to put this argument in `ElectrostaticSolver`.

Reply:
Sure, that makes sense

Reply:
I don't think the adaptive time stepping would be particularly useful for the hybrid code since the allowable timestep in that model is set by the frequency of Whistler waves at the smallest wavelength the grid can support rather than a particle CFL condition.

Reply:
That makes sense!